openedx/core/djangoapps/user_authn/views/register.py

```diff
@@ -341,6 +341,7 @@ def _track_user_registration(user, profile, params, third_party_provider):
             "edx.bi.user.account.registered",
             {
                 'category': 'conversion',
+                'email': user.email,
```

Comment:
Could you please add a code annotation to this line as documented here under example 3: https://open-edx-proposals.readthedocs.io/en/latest/oep-0030-arch-pii-markup-and-auditing.html#docstring-annotations

Reply:
(So is pii_retirement: third_party the right value here?  And is it required to put a space after the dot-dot?)